tests/components/russound_rio/conftest.py

```diff
@@ -0,0 +1,127 @@
+"""Test fixtures for Russound RIO integration."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homeassistant.components.russound_rio.config_flow import FlowHandler
+from homeassistant.components.russound_rio.const import DOMAIN
+from homeassistant.components.russound_rio.media_player import RussoundZoneDevice
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+MOCK_CONFIG = {
+    "host": "127.0.0.1",
+    "port": 9621,
+}
+
+MOCK_DATA = {
+    "host": "127.0.0.1",
+    "port": 9621,
+    "model": "MCA-C5",
+}
+
+
+@pytest.fixture(autouse=True)
+def bypass_setup_fixture():
+    """Prevent setup."""
+    with patch(
+        "homeassistant.components.russound_rio.async_setup_entry", return_value=True
+    ):
+        yield
+
+
+@pytest.fixture(name="mock_config_entry")
+def mock_config_entry_fixture():
```

Comment:
```suggestion
@pytest.fixture
def mock_config_entry() -> MockConfigEntry:
```

Reply:
For the other fixtures, there is no need to append `_fixture` to the name, but this is code style comment